virtual/virtual_inventory

```diff
@@ -21,7 +21,7 @@ virtual-mgmt
 
 [kube-node]
 virtual-mgmt
-virtual-gpu01
+virtual-gpu01 ansible_host=10.0.0.11 ip=10.0.0.11
```

Comment:
Can we put all nodes/IPs in `[all]` for consistency?

Reply:
Yep - moved them in the latest commit.